ONSdigital/eq-submission-confirmation-consumer
# Update github actions workflow and tests

### What is the context of this PR?
Fix failing tests and update github actions to run test suite. 
- Integration tests have been removed from test script. Until a mock service can be introduced. [See here](https://github.com/ONSdigital/eq-submission-confirmation-consumer/pull/12)
- Github action currently fetches the Dummy Notify Api Key as tests would fail otherwise due to a DefaultCredentialsError since GA does not have default credentials set. 
- Dummy Dev Api Key added to GA Secrets

### How to review
- Run tests locally and re run GA's to see if tests are working as expected
- Check if new comments make sense
- Check if new logic for Main.py makes sense (mypy error)
### Checklist
\*[ ] Tests updated
